apps/wing/project-templates/wing/private-api/main.w

```diff
@@ -0,0 +1,20 @@
+bring cloud;
+bring http;
+
+let api = new cloud.Api();
+api.get("/", inflight () => {
+  return {
+    status: 200,
+    body: "hello, from within the VPC!"
+  };
+});
+
+
+let url = api.url;
```

Comment:
why is this needed?

Reply:
Ill check but I dont think its needed, I copy and pasted this template from somewhere, Ill remove